tap_linkedin_ads/client.py

```diff
@@ -94,43 +94,41 @@ def raise_for_error(response):
 
 
 class LinkedinClient(object):
-    def __init__(self, access_token, user_agent=None):
-        self.__access_token = access_token
+    def __init__(self, client_id, client_secret, refresh_token, user_agent=None):
```

Comment:
why not just delete the user_agent, since we will not use it.

Reply:
meh, good point.

Reply:
hmm.. we are actually passing it in in the runner and adding it as a header below